src/mame/misc/marywu.cpp

```diff
@@ -18,10 +18,10 @@
 
 #include "emu.h"
 #include "cpu/mcs51/mcs51.h"
+#include "machine/nvram.h"
 #include "machine/i8279.h"
 #include "sound/ay8910.h"
 #include "speaker.h"
-
 #include "marywu.lh"
```

Comment:
You’ve still removed the blank line separating traditional headers from the generated code for the layout data.  Please put it back.